Content.Shared/Silicons/StationAi/SharedStationAiSystem.cs

```diff
@@ -25,22 +29,27 @@ namespace Content.Shared.Silicons.StationAi;
 public abstract partial class SharedStationAiSystem : EntitySystem
 {
     [Dependency] private   readonly ISharedAdminManager _admin = default!;
-    [Dependency] private   readonly IGameTiming _timing = default!;
-    [Dependency] private   readonly INetManager _net = default!;
-    [Dependency] private   readonly ItemSlotsSystem _slots = default!;
-    [Dependency] private   readonly ItemToggleSystem _toggles = default!;
-    [Dependency] private   readonly ActionBlockerSystem _blocker = default!;
-    [Dependency] private   readonly MetaDataSystem _metadata = default!;
+    [Dependency] private   readonly SharedAirlockSystem _airlocks = default!;
     [Dependency] private   readonly SharedAppearanceSystem _appearance = default!;
+    [Dependency] private   readonly SharedAudioSystem _audio = default!;
+    [Dependency] private   readonly ActionBlockerSystem _blocker = default!;
     [Dependency] private   readonly SharedContainerSystem _containers = default!;
     [Dependency] private   readonly SharedDoorSystem _doors = default!;
+    [Dependency] private   readonly SharedElectrocutionSystem _electrify = default!;
     [Dependency] private   readonly SharedEyeSystem _eye = default!;
-    [Dependency] protected readonly SharedMapSystem Maps = default!;
-    [Dependency] private readonly SharedMindSystem _mind = default!;
+    [Dependency] private   readonly MetaDataSystem _metadata = default!;
+    [Dependency] private   readonly SharedMindSystem _mind = default!;
     [Dependency] private   readonly SharedMoverController _mover = default!;
-    [Dependency] private   readonly SharedTransformSystem _xforms = default!;
+    [Dependency] private   readonly INetManager _net = default!;
+    [Dependency] private   readonly SharedPopupSystem _popup = default!;
+    [Dependency] private   readonly ItemSlotsSystem _slots = default!;
+    [Dependency] private   readonly IGameTiming _timing = default!;
+    [Dependency] private   readonly ItemToggleSystem _toggles = default!;
     [Dependency] private   readonly SharedUserInterfaceSystem _uiSystem = default!;
     [Dependency] private   readonly StationAiVisionSystem _vision = default!;
+    [Dependency] private   readonly SharedTransformSystem _xforms = default!;
+    [Dependency] protected readonly SharedMapSystem Maps = default!;
+    [Dependency] private   readonly SharedPowerReceiverSystem PowerReceiver = default!;
 
```

Comment:
This is a dog's breakfast please just make it alphabetical like it used to be.

Reply:
thats a wierd way to sort it. ok, done.